addons/mrp/models/stock_move.py

```diff
@@ -84,6 +84,22 @@ def _get_aggregated_properties(self, move_line=False, move=False):
         aggregated_properties['line_key'] += f'_{bom.id if bom else ""}'
         return aggregated_properties
 
+    def _compute_product_packaging_qty(self):
+        kit_lines = self.filtered(lambda move_line: move_line.move_id.bom_line_id.bom_id.type == 'phantom')
+        for move_line in kit_lines:
+            move = move_line.move_id
+            bom_line = move.bom_line_id
+
+            # Convert the move line quantity to the product's move uom
+            qty_move_uom = move_line.product_uom_id._compute_quantity(move_line.quantity, move_line.move_id.product_uom)
+            # Convert the product's move uom to the bom line's uom
+            qty_bom_uom = bom_line.product_uom_id._compute_quantity(qty_move_uom, move.product_uom)
+            # calculate the bom's kit qty in kit product uom qty
+            bom_qty_product_uom = bom_line.bom_id.product_uom_id._compute_quantity(bom_line.bom_id.product_qty, move_line.move_id.product_packaging_id.product_uom_id)
```

Comment:
```suggestion
            bom_qty_product_uom = bom_line.bom_id.product_uom_id._compute_quantity(bom_line.bom_id.product_qty, move_line.product_id.uom_id)
```

Reply:
We have to use:
`move_line.move_id.bom_line.bom_id.product_id.uom_id`